src/main/java/com/epam/edumanagementsystem/admin/model/entity/SuperAdmin.java

```diff
@@ -0,0 +1,42 @@
+package com.epam.edumanagementsystem.admin.model.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table
```

Comment:
Add table name in @Table annotation's brackets

Reply:
I didn't use name attribute with the annotation, since i thought it would be ok not to override the default name given by JPA. But for the sake of better readability, I will make the corresponding change.